tests/foreman/api/test_registration.py

```diff
@@ -101,18 +102,27 @@ def test_positive_allow_reregistration_when_dmi_uuid_changed(
 
     :customerscenario: true
 
-    :BZ: 1747177
+    :BZ: 1747177,2229112
 
     :CaseLevel: Integration
     """
     uuid_1 = str(uuid.uuid1())
     uuid_2 = str(uuid.uuid4())
-    rhel_contenthost.install_katello_ca(target_sat)
     target_sat.execute(f'echo \'{{"dmi.system.uuid": "{uuid_1}"}}\' > /etc/rhsm/facts/uuid.facts')
-    result = rhel_contenthost.register_contenthost(module_org.label, lce=ENVIRONMENT)
+    command = target_sat.api.RegistrationCommand(
+        organization=module_org,
+        activation_keys=[module_ak_with_synced_repo.name],
+        location=module_location,
+    ).create()
+    result = rhel_contenthost.execute(command)
     assert result.status == 0
```

Comment:
```suggestion
    assert rhel_contenthost.execute(command).status == 0
```

Reply:
Since it's non-blocking so skipping this.